### ISSUE TITLE
Accessibility: CSVFileName Header Tag

### ISSUE THREAD

**Issue by jaydonkrooss** [tl-its-umich-edu/canvas-course-manager-next#700]
Problematic component: `CSVFileName.tsx`

Issue summary:
- Header tag should be `<p>`: `styled('h5')` root element

Based on these reported problems found in [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

- [Severity: Major] U-M Users Multiple Sections
    - File: filename.csv" should be p not h5
- [Severity: Major] Multiple Non-UM Users
    - After adding a file, "test_um_students_multiple_sections.csv" appears as H5, it should be p instead
- [Severity: Major] Format 3rd Party Gradebook
    - File: filename.csv" should be p not h5
